src/components/atoms/Course.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface courseIdentification {
+    courseNumber: string;
+    courseName: string;
+}
+
+const Course = ({courseNumber, courseName}:courseIdentification) => {
```

Comment:
can you run a linter or use prettier?
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode